docs/en/README.md

```diff
@@ -300,22 +219,4 @@ Otherwise this extension will try to connect to "postgres" Database to check DB
 connection, no matter what you entered in the "Database Name" field during 
 installation.
 
-Make sure you have set the "search_path" correct for your database user. 
-
-## Known Issues
-
-All column and table names must be double-quoted.  PostgreSQL automatically 
-lower-cases columns, and your queries will fail if you don't.
-
-Ts_vector columns are not automatically detected by the built-in search filters.  
-That means if you're doing a search through the CMS on a ModelAdmin object, it 
-will use LIKE queries which are very slow.  
```

Comment:
Is this part no longer true now?

Reply:
Yeah good point. My initial rationale was that a "known issues" list in a README doesn't scale, and should be in Github issues. But re-reading those, they're more highlevel caveats that probably still apply. I'll reinstate them.

Reply:
Those known issues were a duplicate from the main README, this just deletes `docs/en/README.md`